Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -507,6 +508,7 @@ getNearestDrivers mbVariant LatLong {..} radiusMeters merchantId onlyNotOnRide m
       person ^. PersonRole ==. val Person.DRIVER
         &&. person ^. PersonMerchantId ==. val (toKey merchantId)
         &&. driverInfo ^. DriverInformationActive
+        &&. driverInfo ^. DriverInformationMode ==. val (Just DriverInfo.SILENT) ||. driverInfo ^. DriverInformationMode ==. val (Just DriverInfo.ONLINE) ||. Esq.isNothing (driverInfo ^. DriverInformationMode)
```

Comment:
can be written with `whenJust_` instead ?

Reply:
Tried it, whenJust_ is not getting applied on a table column, its used for values i guess